recipes/docker_registry.rb

```diff
@@ -18,6 +18,13 @@
   update_command = "update-ca-certificates"
 end
 
+# On single node hops::dn recipe runs after docker_registry, which causes to miss out the docker recipe.
+#  So we include docker recipe here if not already exists
+docker_home="/srv/hops/docker"
```

Comment:
on single node however, hops::dn gets executed after docker_registry, so added check here to include docker recipe if not exists already.